src/components/engineer/Engineer.js

```diff
@@ -0,0 +1,87 @@
+import React, { useState, createContext } from "react";
+import Plane from "../plane/Plane";
+import EngineerRows from "./EngineerRows";
+import { EngineerFieldsCheck } from "../../helper";
+export const EngineerContext = createContext([]);
+const Engineer = () => {
+  const [engineerrows, setEngineerrows] = useState([]);
```

Comment:
use camel case variables `engineerRows`. 

Instead of adding row components to this, we should create array of engineer objects. 
eg,

[{name:"Kushal",age:22,exp:6 }, ...]

then in the render, loop through the array and create row components